### DISCUSSION TITLE
A shared connection cache is not thread-safe

### DISCUSSION THREAD

**Discussion by 1027663760** [curl/curl#9405] [Q&A] [answered]
https://curl.se/docs/knownbugs.html#A_shared_connection_cache_is_not

When will this bug be fixed? Connection cache is very helpful for performance improvement

**Comment by bagder** [comment, marked as answer]
I'm not aware of anyone working on it. It will take a proper effort to make it working so I suspect it will not happen anytime soon unless someone steps up and helps fund the work.
